forge-gui/res/editions/Modern Horizons 3.txt

```diff
@@ -591,6 +589,14 @@ A29 R A-Guide of Souls @Ryan Valle
 A38 M A-Ocelot Pride @Chris Seaman
 A122 C A-Galvanic Discharge @Zoltan Boros
 
+[full art]
```

Comment:
How come you've moved these to [cards] and duplicated them here too?

Reply:
Oh yes, that' a bug off my script. It was expecting a [full art] section like the one from Bloomburrow.
Thanks for double checking. I have fixed it.